Command/BackupCommand.php

```diff
@@ -51,80 +51,100 @@ protected function initialize(InputInterface $input, OutputInterface $output)
-        }
-        
-        $this->output->write('- <comment>Compressing archive... </comment> ');
-        $processor->compress();
-        $this->output->writeln('<info>OK</info>');
 
-        if (isset($this->storages['dropbox'])) {
-            $this->getContainer()->get('dizda.cloudbackup.client.dropbox')->upload($processor->getArchivePath());
-        }
+            if (isset($this->storages['dropbox'])) {
+                $this->getContainer()->get('dizda.cloudbackup.client.dropbox')->upload($processor->getArchivePath());
+            }
 
-        if (isset($this->storages['google_drive'])) {
-            $this->getContainer()->get('dizda.cloudbackup.client.google_drive')->upload($processor->getArchivePath());
-        }
+            if (isset($this->storages['google_drive'])) {
+                $this->getContainer()->get('dizda.cloudbackup.client.google_drive')->upload($processor->getArchivePath());
+            }
 
-        if (isset($this->storages['cloudapp'])) {
-            $this->getContainer()->get('dizda.cloudbackup.client.cloudapp')->upload($processor->getArchivePath());
-        }
+            if (isset($this->storages['cloudapp'])) {
+                $this->getContainer()->get('dizda.cloudbackup.client.cloudapp')->upload($processor->getArchivePath());
+            }
 
-        if (isset($this->storages['gaufrette'])) {
-            $filesystemName = $this->getContainer()->getParameter('dizda_cloud_backup.cloud_storages')['gaufrette']['service_name'];
+            if (isset($this->storages['gaufrette'])) {
+                $filesystemName = $this->getContainer()->getParameter('dizda_cloud_backup.cloud_storages')['gaufrette']['service_name'];
 
-            $gaufrette = $this->getContainer()->get('dizda.cloudbackup.client.gaufrette');
-            $gaufrette->setFilesystem($this->getContainer()->get($filesystemName));
-            $gaufrette->upload($processor->getArchivePath());
-        }
+                $gaufrette = $this->getContainer()->get('dizda.cloudbackup.client.gaufrette');
+                $gaufrette->setFilesystem($this->getContainer()->get($filesystemName));
+                $gaufrette->upload($processor->getArchivePath());
+            }
 
-        $processor->cleanUp();
-        $this->output->writeln('- <comment>Temporary files have been cleared</comment>.');
+            $processor->cleanUp();
+            $this->output->writeln('- <comment>Temporary files have been cleared</comment>.');
+        }catch(\Exception $e){       
+            $this->output->writeln("");
+            $this->output->writeln("<error>[Exception]</error>");
+            $this->output->writeln("<error>".$e->getMessage()."( code: ".$e->getCode()."; file: ".$e->getFile()."; line: ".$e->getLine().")</error>");
+            
+            if($this->getContainer()->getParameter('dizda_cloud_backup.error_notification')){
+                $container = $this->getContainer();
+                $mailer = $container->get('mailer');
```

Comment:
You can directly call $this->getContainer() rather than save the container into a variable.